src/hooks/useSearchSubgraphByTokenSymbol.ts

```diff
@@ -33,9 +34,15 @@ export const useSearchSubgraphByTokenSymbol = (searchTerm: string) => {
     [searchTerm]
   );
 
+  const ensQuery = ensApi.useResolveNameQuery(
+    searchTerm
+  )
+
+  const isEnsAddress =  ensQuery !== null ? true : false
```

Comment:
`ensQuery !== null` is basically always going to be false because the query object itself will not be null.

I believe this is what we want: `const isEnsAddress = !!ensQuery.data?.address`